File: smiex_api/smiex_api/Models/TipusContenidor.cs
using System;
using System.Collections.Generic;

namespace smiex_api.Models;

public partial class TipusContenidor
{
    public int Id { get; set; }

    public string Tipus { get; set; } = null!;

    public virtual ICollection<Oferte> Ofertes { get; set; } = new List<Oferte>();
}

File: simex_app/app/src/main/java/com/example/simex_app/data/models/TipusContenidor.kt
package com.example.simex_app.data.models

data class TipusContenidor(
    val id: Int,
    val tipus: String,

    // Relación inversa con las Ofertas que usan este contenedor
    // Inicializamos con lista vacía por seguridad
    val ofertes: List<Oferte> = emptyList()
)